### PR TITLE
GET /api/articles/:article_id

### PR DESCRIPTION
**Pull request by Dantren18**

### CODE REVIEW

**Review by DCrawley94 — CHANGES_REQUESTED** [review]
Make to remove all unnecessary console.logs

I'd recommend combining some of your tests e.g. looping through the articles array for this point is kinda overkill - we could just expect using objectContaining and instead of expect.any we can be much more specific

You could also include the assertion for length within the same test block.

Please add some error handling before merging as well

**Comment by Dantren18** [comment]
> Make to remove all unnecessary console.logs
> 
> I'd recommend combining some of your tests e.g. looping through the articles array for this point is kinda overkill - we could just expect using objectContaining and instead of expect.any we can be much more specific
> 
> You could also include the assertion for length within the same test block.
> 
> Please add some error handling before merging as well

Thanks for the feedback Duncan I have implemented your comments now :)

File: __tests__/app.test.js [hunk 1]
```diff
@@ -20,54 +20,143 @@ describe("App", () => {
...
+            {
+              description: "what books are made of",
+              slug: "paper",
+            },
+          ];
+          expect(topics).toEqual(topicsData);
+        });
+    });
+  });
+  describe.only("GET /api/articles/:article_id", () => {
```
Comment (rogersop) [inline __tests__/app.test.js:72]: remove any `.only`s before you merge

**Review by rogersop — APPROVED** [review]
Just the one comment to remove the `.only` then please go ahead and merge 😄

**Pull request merged**